packages/react-core/src/components/ContextSelector/ContextSelectorToggle.tsx

```diff
@@ -10,8 +10,10 @@ export interface ContextSelectorToggleProps {
   id: string;
   /** Classes applied to root element of toggle */
   className?: string;
-  /** Text that appears in the Context Selector Toggle */
-  toggleText?: string;
+  /** Component or string that will be used in the Context Selector.
```

Comment:
Can this be sentence case please.  "Component or string that will be used in the context selector."

Reply:
Yes, I have updated it in the latest commit below :)